src/element.rs

```diff
@@ -46,3 +46,15 @@ implement!(usize);
 
 implement!(c32, c32::new(0.0, 0.0));
 implement!(c64, c64::new(0.0, 0.0));
+
+impl<T> Element for Option<T> where T: Copy + PartialEq {
+    #[inline(always)]
+    fn is_zero(&self) -> bool {
+        self.is_none()
```

Comment:
As I mentioned above--I actually have many scenarios in which there is no such thing as `Some(zero)`, since zero is defined as absence of data. Given that, are you okay with this implementation?